README.md

```
@@ -1,3 +1,7 @@
+I kept up with Dependabot for this project for a very long time, even after I personally stopped using it. The recent advances in AST handling of programming languages have pretty much permanently shifted my attention.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)